### PR TITLE
Add Python 3.8 example using flask and request with pip install in a virtual env

### PR DESCRIPTION
**Pull request by dpfeif**
Aiming at addressing #38 and also explicitly displaying how to add lib dependencies.
Not sure this is according to best practices, couldn't find much info about this in the documentation?

### CODE REVIEW

File: python3.8/config.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "Env": {
+        "FLASK_APP": "hi.py"
+    },
+    "MapDirs": {
+        "./.venv/*": "/.local",
+        "./usr/lib64/*": "/usr/lib/x86_64-linux-gnu"
```
Comment (eyberg) [inline python3.8/config.json:7]: this is fine as an example but it's going to grab everything in that folder which could be massive when in reality you might just be using one or two libs

**Comment by eyberg** [comment]
thanks @dpfeif - can you remove the libraries from the PR so we're not versioning them?

also if it makes sense to create a flask package we can do that

**Comment by dpfeif** [comment]
I spent a bit of time on this to get it working and thought I would share.

**Comment by eyberg** [comment]
thanks!

**Pull request merged**